lib/ex_doc/formatter/html.ex

```diff
@@ -11,6 +11,7 @@ defmodule ExDoc.Formatter.HTML do
   """
   def run(modules, config)  do
     output = Path.expand(config.output)
+    {:ok, _} = File.rm_rf output
```

Comment:
I would use `File.rm_rf!(output)`. If it fails we will get a better reason than badmatch. :)